### PR TITLE
feat: [SRM-10085]: Added time range filter support

### PR DESCRIPTION
**Pull request by shashidhargg**
##### Summary:

Time filter support for the SLO Details page.

##### Jira Links:

https://harness.atlassian.net/browse/SRM-10085

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by shashidhargg** [comment]
trigger cypress

**Review by vasuki-kalluri — APPROVED** [review]

File: src/modules/85-cv/pages/slos/CVSLODetailsPage/DetailsPanel/DetailsPanel.module.scss [hunk 1]
```diff
@@ -30,4 +30,6 @@
 .changesCard {
   width: 100%;
   padding-bottom: 0 !important;
+  box-shadow: none !important;
+  border: 1px solid rgba(11, 11, 13, 0.1) !important;
```
Comment (tan-nhu) [inline src/modules/85-cv/pages/slos/CVSLODetailsPage/DetailsPanel/DetailsPanel.module.scss:34]: Use CSS var name - https://uicore.harness.io/?path=/story/foundation-colors--page&globals=measureEnabled:false

**Review by tan-nhu — CHANGES_REQUESTED** [review]

**Review by tan-nhu — APPROVED** [review]

**Review by vasuki-kalluri — APPROVED** [review]

**Comment by shashidhargg** [comment]
retrigger coverage

**Comment by shashidhargg** [comment]
trigger cypress

**Pull request merged**